charts/spire/examples/production/README.md

```diff
@@ -0,0 +1,36 @@
+# Recommended production setup
+
+> **Note**: You should use all these files from your particular helm chart release. Don't use directly from the upstream
```

Comment:
Avoid using directly?  These are instructions.  Let's avoid being vague.  If possible let's give them the commands, or at least command for "an example" of how to avoid what we're asking them to avoid.